packages/zosfiles/src/methods/create/Create.ts

```diff
@@ -249,35 +249,28 @@ export class Create {
                         break;
 
                     case "dirblk":
-                    // Validate non-zero if dsorg is PS
-                        if (tempOptions.dirblk !== 0 && tempOptions.dsorg === "PS") {
+                        // Validate non-zero if dsorg starts with "PS"
+                        if (tempOptions.dirblk !== 0 && tempOptions.dsorg.startsWith("PS")) {
                             throw new ImperativeError({ msg: ZosFilesMessages.invalidPSDsorgDirblkCombination.message });
                         }
-                        // Validate non-zero if 'dsorg' is PO
-                        if (tempOptions.dirblk === 0 && tempOptions.dsorg === "PO") {
+                        // Validate non-zero if dsorg starts with "PO"
+                        if (tempOptions.dirblk === 0 && tempOptions.dsorg.startsWith("PO")) {
                             throw new ImperativeError({ msg: ZosFilesMessages.invalidPODsorgDirblkCombination.message });
                         }
-
                         break;
 
-                    case "dsntype": {
```

Comment:
Be aware that removal of this code results in the breakage of system test `Create Partitioned Data Set › Expected failures › should fail creating a partitioned data set extended due to invalid data-set-type` in `packages/cli/__tests__/zosfiles/__system__/create/pds/cli.files.create.dataset.pds.system.test.ts`. This test will need to be rewritten or removed if we plan to remove this validation.